src/config/chis-tg/config.json

```diff
@@ -80,7 +178,7 @@
         },
         {
           "friendly_name": "Telephone",
-          "property_name": "telephone",
+          "property_name": "phone",
```

Comment:
This is called `telephone` in contact forms. I think it should change in the contact forms, not here.